src/pages/NewChairConsole.js

```diff
@@ -39,26 +39,55 @@ const getElectionDate = (election) => {
 
 const Records = (props) => {
   //NOTE: currentPage is not zero-indexed
-  const {openModal, currentPage, tableSize, applyTextHover, removeTextHover, openConfirmation} = props;
+  const {openModal, currentPage, tableSize, applyTextHover, removeTextHover, openConfirmation, updateLists} = props;
   const startIndex = (currentPage - 1) * tableSize;
   const endIndex = currentPage * tableSize; //NOTE: endIndex is exclusive, not inclusive
   const visibleWindow = props.filteredList.slice(startIndex, endIndex);
   const dataIDTextStyle = Styles.TABLE.DATA_REQUEST_TEXT;
   const recordTextStyle = Styles.TABLE.RECORD_TEXT;
 
-  const createActionButton = (electionInfo, index) => {
+  const cancelElectionHandler = async (electionData, darId, index) => {
+    const targetElection = electionData.election;
+    const electionId = targetElection.electionId;
+    const electionPayload = Object.assign({}, electionData.election, {status: 'Canceled', finalAccessVote: 'false'});
+
+    try {
+      const updatedElection = await Election.updateElection(electionId, electionPayload);
+      const notification = 'Election has been cancelled';
+      updateLists(updatedElection, darId, index, notification);
+    } catch(error) {
+      Notifications.showError({text: 'Error: Failed to cancel selected Election'});
+    }
+  };
+
+  const createActionButtons = (electionInfo, index) => {
     const name = "cell-button hover-color";
     const e = electionInfo.election;
     const dar = electionInfo.dar;
+    const currentUserId = Storage.getCurrentUser().dacUserId;
+
     if (!isNil(e)) {
       switch (e.status) {
-        case "Open" :
-          const votes = filter({type: "DAC", dacUserId: Storage.getCurrentUser().dacUserId})(electionInfo.votes);
+        case 'Open' :
+          const votes = filter({type: 'DAC', dacUserId: currentUserId})(electionInfo.votes);
+          //Votes can have timestamps stored anywhere between these four key atttributes (result of older data)
+          //Current votes (02/18/2021) store timestamps on updateDate
+          const isFinal = !isEmpty(votes.find(vote => !isNil(vote.createDate) || !isNil(vote.updateDate) || !isNil(vote.lastUpdate) || !isNil(vote.lastUpdateDate)));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6975233)
```suggestion
          const isFinal = !isEmpty(votes.find((vote) => !isNil(vote.createDate) || !isNil(vote.updateDate) || !isNil(vote.lastUpdate) || !isNil(vote.lastUpdateDate)));
```